ci/using_jenkins.md

```diff
@@ -0,0 +1,50 @@
+Using Jenkins
+=============
+
+These are notes on how our Jenkins CI server works.  To that end I'll start by
+noting the server itself is located at http://jenkins-ci.lcrc.anl.gov.  You'll
+need an LCRC account to access it.  When you follow that link you'll be taken to
+our group's dashboard.  On this dashboard you will see the projects Jenkins 
+manages the testing for.  
+
+Adding a New Repo to Jenkins
+----------------------------
+
+Configuring Jenkins occurs primarily when a new repo is added to a GitHub
+organization.  Jenkins automatically scans the GitHub organization for new 
+repos.  When a new repo is found Jenkins looks for a file called `Jenkinsfile`
+in that repo.  This file describes the procedure Jenkins will use to test the
+repo.  For most NWX repos the procedure is quite boilerplate and we have 
+factored that boilerplate out into a script that lives in 
+`DeveloperTools/ci/Jenkins/nwxJenkins.groovy`.  Your `Jenkinsfile` needs to 
+construct a build matrix, load the aforementioned script, and forward the build
+matrix to it.  The script will take care of the rest.  To accomplish this start
+with a `Jenkinsfile` that looks something like:
```

Comment:
This would be a good spot to mention the example in this repo. 

https://github.com/NWChemEx-Project/DeveloperTools/blob/master/ci/Jenkins/Jenkinsfile.example